test/test_autograd.py

```diff
@@ -6188,6 +6192,258 @@ def foo(a):
         self.assertEqual(vhp, torch.mm(v.unsqueeze(0), hes).squeeze(0))
 
 
+class TestAutogradForwardMode(TestCase):
+    def tearDown(self):
+        # Ensure that a failing test won't make others fail
+        while fwAD._current_level >= 0:
+            fwAD.exit_dual_level()
+
+        super().tearDown()
+
+    def test_level_and_packing(self):
+        foo = torch.rand(2)
+        bar = torch.rand(2)
+
+        level = fwAD.enter_dual_level()
+        # For now only level 0 exists
+        self.assertEqual(level, 0)
+
+        # Check that packing works
+        baz = fwAD.make_dual(foo, bar, level=level)
+        baz_primal, baz_tangent = fwAD.unpack_dual(baz, level=level)
+        self.assertEqual(baz_primal, foo)
+        self.assertEqual(baz_tangent, bar)
```

Comment:
Don't you want to assert something more stringent, like that `baz_primal` is a view of `foo`?